test/service/overseas.entities.service.spec.ts

```diff
@@ -1,30 +1,35 @@
 jest.mock("@companieshouse/api-sdk-node");
 jest.mock("@companieshouse/api-sdk-node/dist/services/overseas-entities");
 jest.mock('../../src/utils/logger');
+jest.mock('../../src/utils/application.data');
```

Comment:
See comment below

Reply:
For consistency I will use the double quote, but it is the same here

Reply:
done